src/hooks/use-suggestions.ts

```diff
@@ -666,13 +679,22 @@ export const removeLeadingExpression = (str: string) => {
     return str.match(/\s+(.*)$/)?.[1] || ''
 }
 
-const getValueSuggestions = (dataType: string | string[], operator: string) => {
-    const types: string[] = Array.isArray(dataType) ? dataType : [dataType]
+const getValueSuggestions = (
+    dataType: string | (string | Data)[],
+    operator: string
+) => {
+    const types: (string | Data)[] = Array.isArray(dataType)
+        ? dataType
+        : [dataType]
     const suggestion: string[] = []
 
     if (Array.isArray(dataType)) {
         suggestion.push(
-            ...dataType.filter((type) => !knownDataTypes.includes(type))
+            ...(dataType.filter(
```

Comment:
@makc-dataloop i would prefer to rephrase this to a normal for-of loop and push to the array without the spread operator. 
Spreading of alot of elements can cause a stackoverflow error